apps/plato_single_variant/Readme.md

```diff
@@ -83,27 +86,24 @@ User don't need to specify `load-data` or `load-trait` commands here since app s
 
-
-
 ## What does this app output?
 
 The output of the regression is given as a column-separated file, with one line per model tested, sorted by p-value:
 
-- **Outcome**: Outcome name (if not using the default phenotypic status).
-- **Var1_ID**: IDs for the variables(SNP) of interest.
-- **Var1_Pos**: Chromosome and basepair location of the variant.
-- **Var1_Allele**: Allele encoded for the model.
-- **MAF**: Allele frequency of the encoded allele. Note that his value may be different for the same marker in different models because of different patterns of missingness.
-- **Num_NonMissing**: The number of non-missing samples for the model.
-- **Analysis_Type**: Type of regression performed for each model.
-- **Num_Cases**: Number of samples with case status. It will be set to "0" for linear regression models
-- **Num_Iter**: Number of least-squares steps used before declaring nonconvergence of the method
-- **Converged:** If model was converged. 1=YES;0=NO. Estimates from the model that did not converge are questionable (only relevant to the logistic regression output)
--  **Raw_LRT_pval**: Likelihood ratio test p-value (Full model - Reduced model)
--  **Var1_Pvalue**: Model p-value
--  **Var1_Beta**: Model direction of effect
--  **Var1_SE**: Model Standard Error
--  **Overall_Pval_(LRT)**: Same as Raw LRT p-value. But, sometime in PLATO when a model does not converge it is set to 1 and user can refer to raw LRT p-value but in case of non-convergence the estimates are questionable.
+  - **Outcome**: Outcome name (if not using the default phenotypic status).
+  - **Var1_ID**: IDs for the variables(SNP) of interest.
+  - **Var1_Pos**: Chromosome and basepair location of the variant.
+  - **Var1_Allele**: Allele encoded for the model.
+  - **MAF**: Allele frequency of the encoded allele. Note that his value may be different for the same marker in different models because of different patterns of missingness.
+  - **Num_NonMissing**: The number of non-missing samples for the model.
+  - **Analysis_Type**: Type of regression performed for each model.
+  - **Num_Cases**: Number of samples with case status. It will be set to "0" for linear regression models
+  - **Num_Iter**: Number of least-squares steps used before declaring nonconvergence of the method
+  - **Converged:** If model was converged. 1=YES;0=NO. Estimates from the model that did not converge are questionable (only relevant to the logistic regression output)
+  -  **Raw_LRT_pval**: Likelihood ratio test p-value (Full model - Reduced model)
+  -  **Var1_Pvalue**: Model p-value
+  -  **Var1_Beta**: Model direction of effect
+  -  **Var1_SE**: Model Standard Error
+  -  **Overall_Pval_(LRT)**: Same as Raw LRT p-value. But, sometime in PLATO when a model does not converge it is set to 1 and user can refer to raw LRT p-value but in case of non-convergence the estimates are questionable.
```

Comment:
-  **Overall_Pval_(LRT)**: Same as raw LRT p-value. In the case that the model does not converge, this value is set to 1 and users can refer to raw LRT p-values; however, in cases of non-convergence, the estimates can be questionable.